js/hello-mixit.controller.js

```diff
@@ -7,7 +7,7 @@
 
     function HelloMixitCtrl() {
         this.sayHello = function() {
-            this.greetings = 'Hello World !';
+            this.greetings = "Hello World !";
```

Comment:
Imho it should say `Hello mixit`! And you probably should use single quote like in the rest of the project. We should think about using some formatter.